test/integration/auth.spec.ts

```diff
@@ -1923,6 +1944,9 @@ describe('admin.auth', () => {
         testPhoneNumbers: null,
         smsRegionConfig: deepCopy(expectedUpdatedTenant2.smsRegionConfig),
         recaptchaConfig: deepCopy(expectedUpdatedTenant2.recaptchaConfig),
+        emailPrivacyConfig: {
```

Comment:
same here, please add an object to expectedUpdatedTenant2 and all other occurrence?

Reply:
Explained in the other comment.